bootstrap/bootstrap.go

```diff
@@ -219,6 +219,15 @@ func startServer(
 		signer, err = crypto.NewInMemorySigner(cfg.COAKey, crypto.SHA3_256)
 	case cfg.COAKeys != nil:
 		signer, err = requester.NewKeyRotationSigner(cfg.COAKeys, crypto.SHA3_256)
+	case len(cfg.COACloudKMSKeys) > 0:
+		signer, err = requester.NewKMSKeyRotationSigner(
+			ctx,
+			cfg.COACloudKMSKeys,
+			logger,
+		)
+		if err != nil {
+			return fmt.Errorf("unable to create Cloud KMS key rotation signer: %w", err)
+		}
```

Comment:
no need for this err handling here, it's done after the switch for all

Reply:
Good catch :+1: 
Removed in https://github.com/onflow/flow-evm-gateway/pull/282/commits/ea3bc0a03d1e39a98c65e9f646dd04fac71b7350